src/components/About/TeamGrid.jsx

```diff
@@ -25,9 +25,10 @@ function TeamGrid() {
                 </p>
                 <a href={TeamMembers[0].linkedInLink}>
                   <img
-                    src="https://www.pngall.com/wp-content/uploads/2016/07/Linkedin-Download-PNG.png"
+                    src="https://openvisualfx.com/wp-content/uploads/2019/10/linkedin-icon-logo-png-transparent.png"
```

Comment:
it would be better if you download the icon as an asset and import it locally. The link might stop working again later and we don't want to run into such issues later on

Reply:
The same goes for all icons. Don't use external images. Download the images locally and import them